packages/cashier/src/components/no-balance/__tests__/no-balance.spec.tsx

```diff
@@ -4,21 +4,32 @@ import { createBrowserHistory } from 'history';
 import { Router } from 'react-router';
 import { routes } from '@deriv/shared';
 import NoBalance from '../no-balance';
-
-jest.mock('Stores/connect', () => ({
-    __esModule: true,
-    default: 'mockedDefaultExport',
-    connect: () => Component => Component,
-}));
+import { StoreProvider } from '../../../hooks';
 
 describe('<NoBalance />', () => {
     const history = createBrowserHistory();
+    let mockRootStore;
```

Comment:
```suggestion
    let mockRootStore: DeepPartial<TRootStore>;
```

Reply:
@hamid-deriv By assigning the type I won't be able to make any change inmockRootStore hence I need to pass new amounts during the test, so I didn't assign type here